talisker/endpoints.py

```diff
@@ -120,35 +126,46 @@ def __init__(self, app, namespace='_status'):
             self.urlmap['/metrics'] = 'metrics'
             self.urlmap['/test/prometheus'] = 'test_prometheus'
         if pkg_is_installed('logging-tree'):
-            self.urlmap['/debug/logtree'] = 'debug_logtree'
+            self.urlmap['/info/logtree'] = 'logtree'
+        if pkg_is_installed('psutil'):
+            self.urlmap['/info/workers'] = 'workers'
+        if pkg_is_installed('objgraph'):
+            self.urlmap['/info/objgraph'] = 'objgraph'
 
     def __call__(self, environ, start_response):
         request = Request(environ)
+        response = None
         if request.path.startswith(self.prefix):
             path = request.path[len(self.prefix):].rstrip('/')
             try:
-                funcname = self.urlmap[path]
+                funcname = self.urlmap.get(path, None)
                 func = getattr(self, funcname)
-            except (KeyError, AttributeError):
-                # didn't find /_status endpoint, so pass thru to the app
-                return self.app(environ, start_response)
+            except (KeyError, AttributeError, TypeError):
+                pass
             else:
                 response = func(request)
 
-            return response(environ, start_response)
-        else:
+        if response is None:
+            # pass thru to the app
             return self.app(environ, start_response)
+        else:
+            return response(environ, start_response)
 
     def index(self, request):
         methods = []
-        item = '<li><a href="{0}"/>{1}</a> - {2}</li>'
         for url, funcname in self.urlmap.items():
-            if funcname and url not in self.no_index:
+            if funcname is None:
+                continue
+            if url in self.no_index:
+                continue
+            try:
                 func = getattr(self, funcname)
                 methods.append(
-                    item.format(self.prefix + url, url, func.__doc__))
-        return Response(
-            '<ul>' + '\n'.join(methods) + '<ul>', mimetype='text/html')
+                    link(url, self.prefix + url) + ' - ' + str(func.__doc__),
+                )
+            except AttributeError:
```

Comment:
let's tighten up the `try`..`except` and move `methods.append()` to an `else` clause